rpm/openjpeg.spec

```diff
@@ -68,7 +68,7 @@ popd
 
 %files devel
 %defattr(-,root,root,-)
-%{_includedir}/openjpeg-2.4/
```

Comment:
Can you replace that be defining a macro and then using it:
```spec
%global version_major_minor %{lua: a=rpm.expand('%{version}')\
                                   print(string.match(a, "%d%.%d"))}
```
Doing so removes the need to replace anything besides the version when here's a new version.

Reply:
Probably even easier would be to use wildcard in the file section